src/util/Bundles.js

```diff
@@ -11,19 +11,21 @@
  */
 
 define([
-  'okta',
+  'jquery',
+  'underscore',
+  'handlebars',
   'vendor/lib/q',
   'json!nls/login',
   'json!nls/country',
   'util/Logger',
   'json!config/config',
   'util/BrowserFeatures'
-], function (Okta, Q, login, country, Logger, config, BrowserFeatures) {
+], function ($, _, Handlebars, Q, login, country, Logger, config, BrowserFeatures) {
 
-  var { _, $ } = Okta;
+  // var { _, $ } = Okta;
```

Comment:
Can you help me understand why in other files we use underscore and jquery from `okta`, but here we are explicitly using the dependencies?

Reply:
the `Okta` object is empty at generating `signin-widget` bundle. I think it's because of cycle dependency.
- `courage-for-widget` bundle declares that `shared/util/Bundle` as external
- when integrate with signin widget (webpack.common.config.js), it says `shared/util/Bundle` is alias to `util/Bundle`, which needs to load `okta` (`courage-for-widget.js`), which then asks for alias `util/Bundle` again.